src/Details.js

```diff
@@ -1,11 +1,29 @@
-import React from 'react'
+import React, { Component } from 'react'
 import { Link } from 'react-router-dom';
 
-export default function Details() {
+export default class Details extends Component {
+
+  constructor() {
+    super();
+    this.state = {
+      welcomeMessage: 'This is a WIP :)'
```

Comment:
Please change the state name to message and store Hello, this will be the details page for each Movie & TV show :) in it.

Reply:
Made the amended changes